assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -0,0 +1,183 @@
+          function (data) {
+            successResults.push(data);
+            remaining--;
+            _checkDone();
+          },
+          function (data) {
+            failureResults.push(data);
+            remaining--;
+            _checkDone();
+          }
+        );
+      });
+      return result.promise;
+    }
+    
+    var createResources = function(resources){
+      var titles = {
+        started: "Creating application " + $scope.name + " in project " + $scope.projectName,
+        success: "Created application " + $scope.name + " in project " + $scope.projectName,
+        failure: "Failed to create " + $scope.name + " in project " + $scope.projectName
+      };
+      var helpLinks = {};
+
+      TaskList.add(titles, helpLinks, function(){
+        var d = $q.defer();
+        DataService.createList(resources, $scope)
+          //refactor these helpers to be common for 'newfromtemplate'
+          .then(function(result) {
+                var alerts = [];
+                var hasErrors = false;
+                if (result.failure.length > 0) {
+                  result.failure.forEach(
+                    function(failure) {
+                      var objectName = "";
+                      if (failure.data && failure.data.details) {
+                        objectName = failure.data.details.kind + " " + failure.data.details.id;
+                      } else {
+                        objectName = "object";
+                      }
+                      alerts.push({
+                        type: "error",
+                        message: "Cannot create " + objectName + ". ",
+                        details: failure.data.message
+                      });
+                      hasErrors = true;
+                    }
+                  );
+                } else {
+                  alerts.push({ type: "success", message: "All resource for application " + $scope.name +
+                    " were created successfully."});
+                }
+                d.resolve({alerts: alerts, hasErrors: hasErrors});
+              }
+            );
+            return d.promise;
+          },
+          function(result) { // failure
+            $scope.alerts = [
+              {
+                type: "error",
+                message: "An error occurred creating the application.",
+                details: "Status: " + result.status + ". " + result.data
+              }
+            ];
+          }
+        );
+      Navigate.toProjectOverview($scope.projectName);
+    };
+    
+    var elseShowWarning = function(){
+      $scope.nameTaken = true;
+    };
+    
+    $scope.createApp = function(){
+      var resourceMap = ApplicationGenerator.generate($scope);
+      //init tasks
+      var resources = [];
+      angular.forEach(resourceMap, function(value, key){
+        if(value !== null){
+          Logger.debug("Generated resource definition:" + angular.toJson(value));
```

Comment:
you should be able to do
`Logger.debug("Generated resource definition:", value)`  and then you'll get the inspectable object in the console.  Unless you were specifically wanting to dump the raw json to see what was going to be posted.